libs/auth-api-lib/src/lib/delegations/alive-status.service.ts

```diff
@@ -20,13 +21,22 @@ const decesead = 'LÉST'
 
 @Injectable()
 export class AliveStatusService {
+  alsoUseFakeApi = false
+
   constructor(
     private readonly nationalRegistryClient: NationalRegistryClientService,
     private readonly nationalRegistryV3Client: NationalRegistryV3ClientService,
     private readonly companyRegistryClient: CompanyRegistryClientService,
     @Inject(LOGGER_PROVIDER)
     private logger: Logger,
-  ) {}
+    @Optional()
+    readonly configService?: ConfigService,
+  ) {
+    if (configService) {
+      this.alsoUseFakeApi =
+        configService.get<boolean>('ALSO_USE_FAKE_USER_API') ?? false
+    }
```

Comment:
Same like in the other service and remove L#24

Reply:
Fixed.